JSON_Schemas/Animal_Registrations_and_Movements/iscLicenceCredentialType.json

```diff
@@ -0,0 +1,21 @@
+{
+    "required": [],
+    "type": "object",
+    "allOf": [
+        {
+            "$ref": "icarIdentifierType.json"
+        },
+        {
+            "properties": {
+                "licenseId": {
```

Comment:
This doesn't seem right. So this class will have scheme, id, licenseSchema, licenseId - four members? I thought we could just use "allOf" [{ "$ref": "icarIdentifierType.json" }]  - only special property in here would be a "description" that explained how these are to be used.